apps/native-component-list/src/screens/GL/GLScreens.tsx

```diff
@@ -3,6 +3,7 @@ import './BeforePIXI';
 import * as React from 'react';
 import { Platform } from '@unimodules/core';
 import { Asset } from 'expo-asset';
+import Expo2DContext from 'expo-2d-context';
```

Comment:
```suggestion
import Expo2DContext from 'expo-2d-context';
```
Do we really want to add this lib here? What's it's purpose here? 🤔

Reply:
this is canvas implementation, neither three js nor pixi does not implement it. Bug mentioned in the linked issue was  showing up when using this library, so decided to use it.